python/tvm/contrib/rocm.py

```diff
@@ -1,7 +1,28 @@
 # tvm.contrib.rocm
-
+"""Utility to convert object file to HSA Code Object
+The object file received from LLVM PassManager is relocatable ELF object.
+The routine in this file uses lld to convert it to shared ELF object.
+"""
 from . import util
 
 @tvm.register_func("tvm_callback_rocm_link")
 def callback_rocm_link(obj_bin):
-    return obj_bin
+    """Links object file generated from LLVM to HSA Code Object
+
+    Parameters
+    ----------
+    obj_bin : str
+        The object file
+
+    Return
+    ------
+    cobj_bin : str
```

Comment:
return type should be bytearray. This is important for PackedFunc system in TVM, since str is treated as string with \0 termination, while bytearray are raw bytes